session7/src/com/itschool/course/Curs7Main.java

```diff
@@ -7,25 +7,17 @@
 public class Curs7Main {
 //COURSE 5-6 HOMEWORK
 //EX 1.
-//        public static void main(String[] args) {
-//            String stringOne = "OpenAI";
-//            String stringTwo = "OpenAI";
-//
-//            boolean areSameObject = (stringOne == stringTwo);
-//
-//            System.out.println("Do stringOne and stringTwo refer to the same object? " + areSameObject);
-//        }
-//    }
+        public static void main(String[] args) {
+            String stringOne = "OpenAI";
+            String stringTwo = "OpenAI";
+
+            boolean areSameObject = (stringOne == stringTwo);
+
+            System.out.println("Do stringOne and stringTwo refer to the same object? " + areSameObject);
+        }
+
+
 
-//EX 3.
-//    public static void main(String[] args) {
```

Comment:
don't comment code next time. If you don't need it, just remove it. If you need it, please move it somewhere else (other class or method) uncommented.